frontend/src/app/modules/work-package-graphs/embedded/wp-embedded-graph.component.ts

```diff
@@ -10,6 +10,10 @@ export interface WorkPackageEmbeddedGraphDataset {
   queryId?:number|string;
   groups?:GroupObject[];
 }
+interface ChartDataSet {
+  label:string;
+  data:number[];
+}
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/bf492821197afcb3d2969780842f8fdbddee6382/frontend/src/app/modules/work-package-graphs/embedded/wp-embedded-graph.component.ts#L17"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>